indico/modules/rb_new/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -161,23 +161,20 @@ export default class TimelineItem extends React.Component {
 
         return (
             <Popup trigger={segment} content={popupContent} position="bottom center"
-                   header={reservation && reservation.booked_for_name}
+                   header={reservation && reservation.bookedForName}
                    hideOnScroll />
         );
     };
 
     renderOccurrences = () => {
         const {data} = this.props;
-        return (
-            Object.entries(data).map(([name, occurrences]) => (
-                occurrences.map((occurrence, i) => (
-                    // eslint-disable-next-line react/no-array-index-key
-                    <React.Fragment key={i}>
-                        {this.renderOccurrence(occurrence, classes[name] || 'default', types[name])}
-                    </React.Fragment>
-                ))
-            ))
-        );
+        return Object.entries(data).map(([name, occurrences]) => (
+            occurrences.map((occurrence, i) => (
+                // eslint-disable-next-line react/no-array-index-key
```

Comment:
is this safe? i'd rather use the database ID of the occurrence as a key

Reply:
There may not be a DB id.

Reply:
There may not be a DB id. Anyway, I added the start/end datetimes. I hope that's unique enough.